locales-src/newtab.ftl

```diff
@@ -83,6 +83,13 @@ newtab-menu-delete-history = Delete from History
 newtab-menu-save-to-pocket = Save to { -pocket-brand-name }
 newtab-menu-delete-pocket = Delete from { -pocket-brand-name }
 newtab-menu-archive-pocket = Archive in { -pocket-brand-name }
+newtab-menu-show-privacy-info = Our sponsors & your privacy
+
+## newtab privacy info modal
+newtab-privacy-modal-done = Done
+newtab-privacy-modal-header = Your privacy matters.
+newtab-privacy-modal-paragraph = In addition to dishing up captivating stories, we also show you relevant, highly-vetted content from select sponsors. Rest assured, your browsing data never leaves your personal copy of Firefox—we don’t see it, and our sponsors don’t either.
+newtab-privacy-modal-link = Learn how privacy works on the new tab
```

Comment:
Reset the section comment after the last string by adding a `##`.

Reply:
Like this?

```
newtab-privacy-modal-link = Learn how privacy works on the new tab

##

```

Reply:
Exactly 👍